invenio.cfg

```
@@ -186,6 +186,10 @@ CELERY_BEAT_SCHEDULE = {
             "retry": True,
         },
     },
+    "cleanup-swh-depositions": {
+        "task": "invenio_swh.tasks.cleanup_depositions",
+        "schedule": crontab(minute=0, hour=6), # Every day at 06:00
```

```suggestion
        "schedule": crontab(minute=0, hour=6), # Every day at 06:00 UTC
```

Just to be specific about the timezone